photutils/isophote/tests/test_angles.py

```diff
@@ -0,0 +1,68 @@
+from __future__ import (absolute_import, division, print_function, unicode_literals)
+
+import math
+
+import numpy as np
+import unittest
```

Comment:
OK, I tend to lean toward py.test too, so the package better conforms to the standards used elsewhere. I guess I was just waiting for an excuse to do the change :-)

I will work on that and report here when done.